src/Autocomplete/doc/index.rst

```diff
@@ -171,6 +171,109 @@ After creating this class, use it in your form:
 
 Congratulations! Your ``EntityType`` is now Ajax-powered!
 
+.. _passing-extra-options-to-the-ajax-powered-autocomplete:
+
+Passing Extra Options to the Ajax-powered Autocomplete
```

Comment:
I'd move this block after the standard options.. wdyt ? 

(especially if we rewrite the caution block to add a link to this section)